ui/src/Signin.tsx

```diff
@@ -85,45 +85,30 @@ export class Signin extends PureComponent<Props, State> {
 
       clearInterval(this.intervalID)
 
-      const config = await getAuth0Config()
-      if (CLOUD && config.socialSignUpOn) {
-        // The redirectUri must be the same as the url for the origin of the request
-        // otherwise there's a mismatch and Auth0 cannot validate the response
-        const redirectUri = window.location.href
-        // The responseType is arbitrary as it needs to be a non-empty, non "code" value:
-        // https://auth0.github.io/auth0.js/web-auth_index.js.html#line564
-        const responseType = 'token'
-
-        const auth0 = new auth0js.WebAuth({
-          domain: config.domain,
-          clientID: config.clientID,
-          redirectUri,
-          responseType,
-        })
-        // This is the result of JS & Auth0 weirdness
-        return new Promise((resolve, reject) => {
-          // The TLDR is that checkSession is not awaiting the callback to complete
-          // So checkSession can return a successful response and continue with the operation
-          // without the callback being completely finished
-          return auth0.checkSession({}, (error, webResponse) => {
+      if (CLOUD) {
+        const config = await getAuth0Config()
+        if (config.socialSignUpOn) {
+          // The responseType is arbitrary as it needs to be a non-empty, non "code" value:
+          // https://auth0.github.io/auth0.js/web-auth_index.js.html#line564
+          const auth0 = new auth0js.WebAuth({
+            domain: config.domain,
+            clientID: config.clientID,
+            state: config.state,
+            redirectUri: config.redirectURL,
+            responseType: 'token',
+          })
+
+          auth0.checkSession({}, error => {
```

Comment:
@ischolten should we return here? It seems like what'll end up happening is that we'll continue onward after this operation finishes, and we'll simply call `handleSignin` on the API? I'm also not sure how relevant these changes are now that we're going to be redirecting from IDPE rather than the UI